docs/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Closes (issue name)#(number).
```

Comment:
Unfortunately this doesn't automatically close the issue, it has to be just "Closes #<number>". GitHub does show you the issue when you hover over the number tho which is nice enough :)